### ISSUE TITLE
Thai Gov mobile application hall-of-fame and graveyard

### ISSUE THREAD

**Issue by p16i** [codeforthailand/welcome#5]
Inspired by https://killedbygoogle.com.

Over the past years, there are many applications developed by the government and its organisations. However, because of being applications, these applications lack of accessibility and maintenance; hence, they are outdated and unusable when mobile OSs are upgraded.  

It would be interesting to collect the data about the developments of these applications. Below are variables that we might collect:

```
Name:
Organization:
Description:
Budget: 
Period of active: 
```

**Comment by unnawut** [comment]
Would also be great to put successful apps on hall of fame. We should encourage those who've done well too.

E.g. I'm pretty satisfied with the Revenue Department's web app for personal income tax reporting, as well as the export/import of encrypted stock dividend tax credit data from TSD to the Revenue Department.

**Comment by p16i** [comment]
Yes, I totally agree on that. We need to find a way to measure whether each app is successful. One way might be looking at when its recent release date, but that's is very crude. 

Yesterday, I found that there is only one app that isn't downloadable from ~20 apps that I collected.
Please find more information at: https://docs.google.com/spreadsheets/d/1kfYEn8RHeEKWMnqPgMwatoDfc67LN1kxRDJYfZ-tAGA/edit?usp=sharing
 

List of all applications developed by government organisations:
https://www.egov.go.th/th/e-government-service/435/
